cabby/geo/map_processing/map_structure.py

```diff
@@ -39,7 +39,8 @@
 # This variable is used (:map_structure.py; :edge.py; cabby.geo.util.py,
 # cabby.geo.walk.py) to project the geometries into this CRS for geo operations
 # such as calculating the centroid.
-OSM_CRS = 32633
+OSM_CRS = 4326
+POI_DISTANCE = 50
```

Comment:
Please comment this module-level variable as well.

Reply:
removed. This isn't needed after all